common/changes/@microsoft/api-extractor/sunilsurana-issue912_2018-10-30-05-58.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Fix for issue 912. Adding lib support to read json config file.",
```

Comment:
>"Fix for issue 912. Adding lib support to read json config file.", [](start = 17, length = 66)

Please see this doc for some guidance regarding change log wording:
[https://rushjs.io/pages/best_practices/change_logs/](https://rushjs.io/pages/best_practices/change_logs/)

Reply:
Fixed. Changed the comment.